docs/update_and_migration/migrate_to_ibexa_dxp/migrating_from_ez_publish_platform.md

```diff
@@ -242,6 +242,70 @@ Import to your database the changes provided in one of the following files. It's
 
 Steps here should only be done once you are ready to move away from legacy and Legacy Bridge, as the following Field Types are not supported by legacy. In other words, content you have migrated will not be editable in legacy admin interface anymore, but rather in the more modern eZ Platform back-end UI, allowing you to take full advantage of what the Platform has to offer.
 
+The Field Types unsupported in eZ Platform are:
+
+- [XmlText](#321-migrate-xmltext-content-to-richtext)
+- [Page](#322-migrate-page-field-to-page-ez-enterprise-only)
+- Star Rating
+- Matrix
```

Comment:
The matrix fieldtype is still supported as mentioned in "old" doc : https://github.com/ezsystems/developer-documentation/pull/1335/files#diff-c5a23f68142e5e2271c509d89f7a5e9d82d06d6c0fdf629b8fa509613e2bf0b2

However, in order to make it run on later 2.5 versions you need to run a migration script : https://doc.ibexa.co/en/latest/update_and_migration/from_1.x_2.x/update_db_to_2.5/#changes-to-matrix-field-type